src/domain/__tests__/SipLine.test.js

```diff
@@ -1,13 +1,34 @@
-import SipLine from '../SipLine';
+import SipLine, { MINIMUM_WAZO_ENGINE_VERSION_FOR_PJSIP } from '../SipLine';
+import Session from '../Session';
 
 describe('SipLine domain', () => {
-  it('can tell if we have videoConference capabilities', () => {
-    const getSipLine = (options) => new SipLine({ options });
-
-    expect(getSipLine([['max_audio_streams', '1'], ['max_video_streams', '2']]).hasVideoConference()).toBe(true);
-    expect(getSipLine([['max_audio_streams', '1'], ['max_video_streams', '1']]).hasVideoConference()).toBe(false);
-    expect(getSipLine([['max_audio_streams', '2']]).hasVideoConference()).toBe(false);
-    expect(getSipLine([['max_video_streams', '2']]).hasVideoConference()).toBe(false);
-    expect(getSipLine([['max_audio_streams', '2'], ['max_video_streams', '2']]).hasVideoConference()).toBe(true);
+  it('CHAN_SIP : can tell if we have videoConference capabilities', () => {
```

Comment:
why CHAN_SIP?

Reply:
Because I want to be sure the sdk still support engine older than 20.13. There's maybe a better name to make a distinction between PJSIP endpoint and previous endpoint.

Any suggestions ? 😊

Reply:
This option doesn't exist in chan_sip, it's only PJSIP and we are using using PJSIP for a long time now. So I'm not sure to understand what you want to do here.

Reply:
Bad understanding from myself, renamed differently the tests cases :)